scss/_variables.scss

```diff
@@ -684,6 +684,8 @@ $line-height-base:            calc(18 / 16) !default;
 $line-height-sm:              calc(16 / 14) !default;
 $line-height-lg:              calc(30 / 16) !default;
 // stylelint-enable function-disallowed-list
+$small-line-length:           32ch !default;
```

Comment:
Separate clearly this block from the others by adding an empty line before `$small-line-length` and after `$medium-line-length`.

Those variables shouldn't be named `$line-length-sm` and `$line-length-md` or include the concepts of paragraphs or max-width?